rpc/README.md

```diff
@@ -1908,9 +1918,52 @@ http://localhost:8114
                     "score": "0x64"
                 }
             ],
+            "connected_duration": "0x2f",
             "is_outbound": true,
+            "last_ping_duration": "0x1a",
             "node_id": "QmXwUgF48ULy6hkgfqrEwEfuHW7WyWyWauueRDAYQHNDfN",
-            "version": "0.31.0 (4231360 2020-04-20)"
+            "protocols": [
+                {
+                    "id": "0x4",
+                    "version": "0.0.1"
+                },
+                {
+                    "id": "0x2",
+                    "version": "0.0.1"
+                },
+                {
+                    "id": "0x1",
+                    "version": "0.0.1"
+                },
+                {
+                    "id": "0x64",
+                    "version": "1"
+                },
+                {
+                    "id": "0x6e",
+                    "version": "1"
+                },
+                {
+                    "id": "0x66",
+                    "version": "1"
+                },
+                {
+                    "id": "0x65",
+                    "version": "1"
+                },
+                {
+                    "id": "0x0",
+                    "version": "0.0.1"
+                }
+            ],
+            "sync_state": {
+                "best_known_header_hash": null,
```

Comment:
Suggested add: `task_number`, it allows us to see the quality of service of the node during the synchronization period

Reply:
Do you mean the `DownloadScheduler#task_count`?

Reply:
yes

Reply:
Got, will add it after https://github.com/nervosnetwork/ckb/pull/2190 is merged, need to resolve conflict first...

Reply:
@driftluo added `can_fetch_count` / `inflight_count`, please review again